modules/mono/glue/GodotSharp/GodotSharp/Core/NativeInterop/ExceptionUtils.cs

```diff
@@ -99,10 +99,41 @@ private static void SendToScriptDebugger(Exception e)
             }
         }
 
+        private static void InvokeUserUnhandledExceptionReporter(Exception e)
```

Comment:
The name is not correct: the reporter should be invoked only in regular exceptions. Users can manage the unhandled exceptions using `AppDomain.CurrentDomain.UnhandledException`  which is exactly what GodotSharp is doing here: https://github.com/godotengine/godot/blob/master/modules/mono/glue/GodotSharp/GodotSharp/Core/GodotUnhandledExceptionEvent.cs